cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,16 @@
+class TimeLoggingPage{
+    getCalendar(){
+        return cy.get('.calendar')
+    }
+
+    getTileForm(){
+        return cy.get('.tile.form')
+    }
+
+    getTodaysDay(){
+        return cy.get('.calendar__date').contains(new Date().getDate())
+    }
```

Comment:
this function actually validates as well, not only getting it

Reply:
from your use case it does not seems like you are using what this function returns so maybe there is no need to return anything (remove unnecessary "return" )